bridge_adaptivity/module/routing.py

```diff
@@ -0,0 +1,8 @@
+# chat/routing.py
```

Comment:
`chat` again, please clear the code from the odd artifacts, so we can continue reviewing process.